### PR TITLE
LPS-61153 Portal Directory advanced filters Country and Region in Organization section are not available for guests

### PR DESCRIPTION
**Pull request by dacousalr**
Hi @topolik,

Taking into account what @adolfopa and you discussed by e-mail, here is the original solution for this issue "reloaded".

Thanks

### CODE REVIEW

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -3357,7 +3357,9 @@
     # Set a list of comma delimited origins that will not be checked for an
     # authentication token.
     #
-    #auth.token.ignore.origins=\
+    auth.token.ignore.origins=\
+         com.liferay.portal.jsonwebservice.JSONWebServiceServiceAction:/country/get-countries,\
+         com.liferay.portal.jsonwebservice.JSONWebServiceServiceAction:/region/get-regions
```
Comment (topolik) [inline portal-impl/src/portal.properties]: Do we need this?

With this whitelist the services can be read from 3rd party sites

Comment (dacousalr) [inline portal-impl/src/portal.properties]: I have re-checked this and you are absolutely right. Adding these services to that whitelist is not necessary at all.

**Comment by dacousalr** [comment]
ci:retest

**Comment by topolik** [comment]
Pull request submitted at: https://github.com/brianchandotcom/liferay-portal/pull/40071
View total diff: https://github.com/topolik/liferay-portal/compare/70d07e8324...c8b5b744e2

**Pull request closed without merging**